pages/become_a_member.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div class="max-w-7xl mx-auto spx-6 px-6 lg:px-8 text-lg">
+        <h1 class="text-2xl font-bold py-4">Become a Member</h1>
+        <div class="text-gray-500">
+            <p class="py-2">
+                Thank you for your interest in becoming a DES member! Firstly we
+                would like to point out that
+                <strong>
+                    you do not need to be a member to use the DES Slack, attend
+                    meetups and events, or to join the monthly society meeting
+                    and AGM.
+                </strong>
+            </p>
+            <p class="py-2">Any member of the public is welcome to:</p>
+            <div class="py-2">
+                <ul class="list-disc ml-10">
+                    <li>Join the Edmonton developer community</li>
+                    <li>Chat on our Slack</li>
+                    <li>Attend DES supported meetups and events</li>
+                    <li>Come out to DES monthly planning meetings</li>
+                    <li>Participate in Working Groups</li>
+                </ul>
+            </div>
+            <p class="py-2">Membership is only required to:</p>
+            <div class="py-2">
+                <ul class="list-disc ml-10">
+                    <li>Vote on society business</li>
+                    <li>Borrow DES equipment (cameras, mics, etc.)</li>
+                    <li>
+                        Use the DES resources (like our YouTube channel or
+                        streaming tools)
+                    </li>
+                    <li>Start an event or group officially supported by DES</li>
+                    <li>Use the DES name or branding</li>
+                    <li>Champion a Working Group</li>
+                    <li>
+                        Give us $20/year to support the work we do (thank you!)
+                    </li>
+                </ul>
+            </div>
+            <p class="py-2">
+                If you're still interested in becoming a member, please complete
+                the form below and then e-transfer $20 to
+                memberships@devedmonton.com. Membership fee's are to be paid
+                annually (we will remind you 😛 ) and if you have any questions
+                about when your membership expires, these can be answered from
```

Comment:
Last pr we were worried about the wording of this, so I am hoping just saying refer to the bylaws will hopefully get us through :P